addon/package.json

```diff
@@ -57,5 +74,19 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "ember-addon": {
+    "main": "addon-main.js",
+    "type": "addon",
+    "version": 2,
+    "app-js": {
+      "./initializers/ember-engines-router-service.js": "./dist/_app_/initializers/ember-engines-router-service.js",
+      "./utils/namespace-engine-route-name.js": "./dist/_app_/utils/namespace-engine-route-name.js",
+      "./utils/resembles-url.js": "./dist/_app_/utils/resembles-url.js",
+      "./utils/root-owner.js": "./dist/_app_/utils/root-owner.js"
+    }
+  },
+  "volta": {
```

Comment:
I don't want to use volta, I want to keep this addon agnostic from Node version management ; )

Reply:
removed it